src/lm/arpa-file-parser.cc

```diff
@@ -29,9 +29,19 @@
 
 namespace kaldi {
 
+void ArpaParseOptions::Register(OptionsItf *opts) {
```

Comment:
We normally define Register inline in the header because it's effectively part of the documentation.
A comment mentioning why not all the members are registered would be good.